Makefile

```diff
@@ -1,11 +1,12 @@
 # THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
 #
-# Generated on 2023-06-04T07:59:54Z by kres 3b7a14e.
+# Generated on 2023-07-03T14:32:42Z by kres 614da92.
 
 # common variables
 
 SHA := $(shell git describe --match=none --always --abbrev=8 --dirty)
 TAG := $(shell git describe --tag --always --dirty)
+ABBREV_TAG := $(shell git describe --tag --always --match v[0-9]\* --abbrev=0 )
```

Comment:
I think it will be easier to move `undefined` fallback here

Reply:
Done